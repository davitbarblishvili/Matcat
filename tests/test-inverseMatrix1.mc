func main() int 
{
    matrix a; 
    matrix c; 
    a = [[2,5],[5,1]];
    inv(a);
    
    return 0;

}