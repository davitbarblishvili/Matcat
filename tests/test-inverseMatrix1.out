(
[-0.04 0.22 ]
[0.22 -0.09 ]
)